acceptable/_service.py

```diff
@@ -80,7 +80,19 @@ def clear(self):
         self._current_version = None
 
 
-Metadata = APIMetadata()
+__metadata = None
```

Comment:
This looks a bit like an attempt to invoke the private name-mangling feature, but that only works in classes.  Maybe just `_metadata`?